Documentation/operations/upgrade.rst

```diff
@@ -330,13 +303,393 @@ Annotations:
   Users may opt to disable mTLS by using the following Helm options when
   upgrading (strongly discouraged):
 
-  - ``global.hubble.tls.enabled=false``
-  - ``global.hubble.tls.auto.enabled=false``
+  - ``hubble.tls.enabled=false``
+  - ``hubble.tls.auto.enabled=false``
 * Cilium has upgraded its CRDs to v1, from v1beta1. Users must run the
   pre-flight checker mentioned above. The pre-flight check will ensure the
   custom resources installed inside the cluster are well-formed.
 * The Cilium agent is now enforcing API rate limits for certain API calls. See
   :ref:``api_rate_limiting`` for more information.
+* Cilium Helm charts have been completely re-factored. Most of the values used
+  to drive Helm charts have been re-scoped from global values to be part of a
+  single `Cilium`_, Helm chart. When upgrading from a previous version of Cilium,
+  the values will need to be provided using the new structure. In most cases the
+  prefixes of ``global.``, ``agent.``, and ``config.`` can be dropped from the
+  previously used value name. As an example, if you previously ran the command
+  ``helm install --set global.ipv4.enabled=true`` you would now run ``helm
+  install --set ipv4.enabled=true``. See the following table which calls out
+  specific values where the prefix cannot be simply dropped, followed by a full
+  table of old and new Helm values.
+
+  The following values have been renamed:
+
++----------------------------------------------+--------------------------------------------+
+| <= v1.8.x Value                              | >= 1.9.x Renamed Value                     |
++==============================================+============================================+
+| agent.enabled                                | agent                                      |
++----------------------------------------------+--------------------------------------------+
+| config.bpfMasquerade                         | bpf.masquerade                             |
++----------------------------------------------+--------------------------------------------+
+| config.bpfClockProbe                         | bpf.clockProbe                             |
++----------------------------------------------+--------------------------------------------+
+| config.bpfTProxy                             | bpf.tproxy                                 |
++----------------------------------------------+--------------------------------------------+
```

Comment:
Oh, and this one is new in v1.9 so we don't need to list the value in the upgrade docs.